detect/errorloaderdetector.py

```diff
@@ -0,0 +1,33 @@
+import pandas as pd
+
+from .detector import Detector
+
+
+class ErrorsLoaderDetector(Detector):
```

Comment:
Can we add a unit test as well...
Set up the test case with a temp file created using `tempfile` [as documented here](https://docs.python.org/3.7/library/tempfile.html) and manually write a couple of violations. Write one valid and one invalid with expected exception raise

Reply:
Will be added in the test pr.